csharp/examples/FlightClientExample/Program.cs

```diff
@@ -0,0 +1,134 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Threading.Tasks;
+using Grpc.Net.Client;
+using Apache.Arrow.Flight.Client;
+using Apache.Arrow.Flight;
+using Apache.Arrow;
+using System.Linq;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.CompilerServices;
+using System.Collections.Generic;
+
+namespace FlightClientExample
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            string host = args.Length > 0 ? args[0] : "localhost";
+            string port = args.Length > 1 ? args[1] : "433";
+
+            CancellationTokenSource source = new CancellationTokenSource();
```

Comment:
Do we ever cancel this `CancellationTokenSource`? If not, can it be removed?

Reply:
Is it optional? I remember `ResponseStream.MoveNext` required a cancellation token so I created one here. Or maybe there's a simpler way to pass a token?

Reply:
If you aren't ever going to cancel the operation, you can just pass in `default` or `CancellationToken.None`